executor/api/kafka/server.go

```diff
@@ -29,11 +31,12 @@ const (
 )
 
 const (
-	ENV_KAFKA_BROKER       = "KAFKA_BROKER"
-	ENV_KAFKA_INPUT_TOPIC  = "KAFKA_INPUT_TOPIC"
-	ENV_KAFKA_OUTPUT_TOPIC = "KAFKA_OUTPUT_TOPIC"
-	ENV_KAFKA_FULL_GRAPH   = "KAFKA_FULL_GRAPH"
-	ENV_KAFKA_WORKERS      = "KAFKA_WORKERS"
+	ENV_KAFKA_BROKER            = "KAFKA_BROKER"
+	ENV_KAFKA_INPUT_TOPIC       = "KAFKA_INPUT_TOPIC"
+	ENV_KAFKA_OUTPUT_TOPIC      = "KAFKA_OUTPUT_TOPIC"
+	ENV_KAFKA_FULL_GRAPH        = "KAFKA_FULL_GRAPH"
+	ENV_KAFKA_WORKERS           = "KAFKA_WORKERS"
+	ENV_KAFKA_SECURITY_PROTOCOL = "KAFKA_SECURITY_PROTOCOL"
```

Comment:
:wrench: I don't believe this is used anywhere now (which makes sense, as the env var handling has moved to `utils`